### PR TITLE
Jv 131 mobile friendly credits

### LINKED ISSUE #131: Mobile friendly credits screen
## Expected Behavior
Some elements in the credits screen are not placed correctly in mobile screen sizes.

## Actual/Current Behavior
The devs title is not centered.
Some of the Pokemon need to be resized.

### PR DESCRIPTION
**Pull request by Wolftownn**
## Changes
1. Adds percentages to elements in the credits screen to make them mobile friendly.

## Purpose
Makes the app look better in mobile screens.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #131

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Great fix! Looks good in app

**Comment by krferrell** [comment]
I was not added as a reviewer. I'm offended.

**Review by Gjhnsn — APPROVED** [review]
Very nice adjustments with the percentages

**Review by BlackCubes — APPROVED** [review]
Nice job on making it work on smaller devices!

File: pokemon-app/src/styles/CreditScreen.css [hunk 1]
```diff
@@ -87,7 +88,7 @@
     height: 23px;
     margin-top: -2px;
 }
-.whole-noct{
+.wholeNoct{
```
Comment (krferrell) [inline pokemon-app/src/styles/CreditScreen.css:91]: Why did you move away from the hyphen case?

Comment (Wolftownn) [reply pokemon-app/src/styles/CreditScreen.css:91]: I noticed I had named it without a hyphen in my earlier code haha

**Review by dannysanchez559 — APPROVED** [review]
Dialed in that css

File: pokemon-app/src/styles/CreditScreen.css (hunk 1 above)
Comment (kmausolf) [reply pokemon-app/src/styles/CreditScreen.css:91]: I think the hyphen case is better, or at least for consistency's sake.  At BWTC, we only use the hyphen case in css.

**Review by gonzalespaulb — APPROVED** [review]
last but not least. great job fellas

File: pokemon-app/src/styles/CreditScreen.css (hunk 1 above)
Comment (Wolftownn) [reply pokemon-app/src/styles/CreditScreen.css:91]: Ok, thanks Kendrick! I'll make sure to stick to hyphens from now on.
